lib/sparse/src/common/sparse_vector.rs

```diff
@@ -85,7 +85,55 @@ impl SparseVector {
             None
         }
     }
+
+    /// Construct a new vector that is the result of performing all indices-wise operations
+    pub fn perform_operation(
```

Comment:
please basic unit tests for it :)

sum and multiplication

Reply:
added basic sum test